modules/tinymce/src/core/main/ts/keyboard/InlineBoundariesNavigation.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Fun } from '@ephox/katamari';
+import Editor from '../api/Editor';
+import * as Settings from '../api/Settings';
+import CaretPosition from '../caret/CaretPosition';
+import * as BoundaryLocation from './BoundaryLocation';
+import * as BoundarySelection from './BoundarySelection';
+import * as InlineUtils from './InlineUtils';
+import * as NavigationUtils from './NavigationUtils';
+
+const isInlineBoundaries = (editor: Editor) => {
```

Comment:
This function would not be needed if you use the change I provided below.

Reply:
updated